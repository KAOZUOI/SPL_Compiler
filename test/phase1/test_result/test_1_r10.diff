1,2c1
< Error type B at Line 1: Missing semicolon ';'
< Error type B at Line 5: Missing semicolon ';'
---
> Error type B at Line 4: Missing specifier
