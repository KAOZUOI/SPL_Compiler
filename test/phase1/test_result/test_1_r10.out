Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test_1_r10
        LP
        RP
      CompSt (2)
        LC
        StmtList (3)
          Stmt (3)
            Def (3)
              Specifier (3)
                TYPE: int
              DecList (3)
                Dec (3)
                  VarDec (3)
                    ID: a
                COMMA
                DecList (3)
                  Dec (3)
                    VarDec (3)
                      ID: b
                    ASSIGN
                    Exp (3)
                      INT: 1
              SEMI
          StmtList (4)
            Stmt (4)
              Exp (4)
                Exp (4)
                  ID: a
                ASSIGN
                Exp (4)
                  INT: 10
              SEMI
            StmtList (5)
              Stmt (5)
                Def (5)
                  Specifier (5)
                    TYPE: int
                  DecList (5)
                    Dec (5)
                      VarDec (5)
                        ID: i
                      ASSIGN
                      Exp (5)
                        INT: 0
                  SEMI
              StmtList (6)
                Stmt (6)
                  WHILE
                  LP
                  Exp (6)
                    Exp (6)
                      ID: i
                    LT
                    Exp (6)
                      ID: a
                  RP
                  Stmt (7)
                    CompSt (7)
                      LC
                      StmtList (8)
                        Stmt (8)
                          Exp (8)
                            Exp (8)
                              ID: b
                            ASSIGN
                            Exp (8)
                              Exp (8)
                                ID: b
                              PLUS
                              Exp (8)
                                ID: i
                          SEMI
                        StmtList (9)
                          Stmt (9)
                            Exp (9)
                              Exp (9)
                                ID: i
                              ASSIGN
                              Exp (9)
                                Exp (9)
                                  ID: i
                                PLUS
                                Exp (9)
                                  INT: 1
                            SEMI
                      RC
        RC
